src/jazkarta/zoterolib/content/zotero_library.py

```diff
@@ -114,6 +115,30 @@ def fetch_items(self, start=0, limit=100):
             else:
                 current_batch = []
 
+    def get_most_recent_obj_date(self):
+        """Return a string representing the most recent modification date of an object in this library."""
+        catalog = getToolByName(self, "portal_catalog")
+        most_recent_objs = catalog.searchResults(
+            sort_on='modified',
+            portal_type="ExternalZoteroItem",
+            path='/'.join(self.getPhysicalPath()),
```

Comment:
Should probably include `sort_order='descending'` and `sort_limit=1` as an optimization, then the check below should be on `most_recent_objs[0]`.